lib/JSON/CalDAV/CalendarHandler.php

```
@@ -0,0 +1,476 @@
+        foreach ($calendars as $calendar) {
+            if ($calendar instanceof \ESN\CalDAV\SharedCalendar && !$calendar->isSharedInstance() && $calendar->isPublic()) {
+                $items[] = $this->calendarToJson($nodePath . '/' . $calendar->getName(), $calendar, $withRights);
+            }
+        }
+
+        return $items;
+    }
+
+    public function getCalendarInformation($nodePath, $node, $withRights) {
+        $baseUri = $this->server->getBaseUri();
+        $requestPath = $baseUri . $nodePath . '.json';
```

PHPMD (2.15.0)

Avoid unused local variables such as '$requestPath'. (undefined)

(UnusedLocalVariable)